turk-one/.idea/jsLibraryMappings.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="JavaScriptLibraryMappings">
+    <file url="PROJECT" libraries="{meteor-packages-auto-import-browser}" />
```

Comment:
He guys, i’m not sure if I will have time to look at the code. So maybe if master builder Sahand approves than i’m also cool. 
We are going to deploy this after the experiment (probably Wednesday anyway)

@Sahand, I kind of need to know how to load up your backups into a local database. Did you already try this?
also if you haven’t done so already please back up the db. 

Thanks

> On 07 Dec 2015, at 15:49, Fernando Spanghero <notifications@github.com> wrote:
> 
> In turk-one/.idea/jsLibraryMappings.xml <https://github.com/sahandnayebaziz/SDCL-Turk-One/pull/11#discussion_r46896541>:
> 
> > @@ -1,6 +1,7 @@
> >  <?xml version="1.0" encoding="UTF-8"?>
> >  <project version="4">
> >    <component name="JavaScriptLibraryMappings">
> > +    <file url="PROJECT" libraries="{meteor-packages-auto-import-browser}" />
> ops, sorry! I remember Edgar mentioned that.
> 
> yeah, let's update .gitignore.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/sahandnayebaziz/SDCL-Turk-One/pull/11/files#r46896541>.
>